src/main/scala/com/mwz/sonar/scala/pr/GithubPrReviewJob.scala

```diff
@@ -147,7 +147,7 @@ object GithubPrReviewJob {
   // TODO: This is quite grim.
   def allCommentsForIssues(
     issues: Map[InputFile, List[Issue]],
-    mappedPatches: Map[String, ErrorOr[Map[FileLine, PatchLine]]],
```

Comment:
Maybe the type alias is not longer required?

Reply:
Maybe, but I haven't done much error handling yet so I'll hold off with removing it until later if it turns out it's not used anywhere.